demos/mixed_wave/demo_RTwave_PEP.py.rst

```diff
@@ -0,0 +1,112 @@
+:orphan:
```

Comment:
Why do we need this tag?

Unrelated, we should probably edit `docs/source/index.rst` to add this demo to the index